examples/erc20-react-example/src/App.tsx

```diff
@@ -217,86 +190,102 @@ function App() {
   }, [data, logicConnected]);
 
   useEffect(() => {
-    if (metaIsConnected && sygmaInstance !== undefined) {
-      handleConnect();
-      getAccountData(sygmaInstance! as Sygma);
-      setValue("from", sygmaInstance.bridgeSetup?.chain1.domainId!)
-      setValue("to", sygmaInstance.bridgeSetup?.chain2.domainId!)
+    if (
+      metaIsConnected &&
+      sygmaInstance !== undefined &&
+      sygmaInstance.bridgeSetup
+    ) {
+      void handleConnect();
+      void getAccountData(sygmaInstance);
+      setValue("from", sygmaInstance.bridgeSetup?.chain1.domainId);
+      setValue("to", sygmaInstance.bridgeSetup?.chain2.domainId);
     }
   }, [metaIsConnected]);
 
-  const submit = async (values: any) => {
-    const { amount, address, from, to } = values;
-
-    const basicFeeData = await (sygmaInstance as Sygma).fetchBasicFeeData(
-      {
+  const submit = async (values: {
+    amount: string;
+    address: string;
+    from: string;
+    to: string;
+  }): Promise<void> => {
+    const { amount, address } = values;
+    if (sygmaInstance) {
+      const basicFeeData = await sygmaInstance.fetchBasicFeeData({
         amount: amount,
         recipientAddress: address,
-      }
-    );
-    if (!(basicFeeData instanceof Error)) {
-      console.log(
-        "🚀 ~ file: App.tsx ~ line 244 ~ submit ~ feeOracleData",
-        basicFeeData
-      );
-      if (
-        window.confirm(
-          `Current fee: ${basicFeeData.calculatedRate} ${basicFeeData.erc20TokenAddress === constants.AddressZero ? 'ETH' : basicFeeData.erc20TokenAddress}\n\nTotal(amount+fee): ${
-            parseFloat(amount) + parseFloat(basicFeeData.calculatedRate)
-          } tokens\n\nDo you really want to proceed?`
-        )
-      ) {
-        const approveTx = await (sygmaInstance as Sygma).approve({
-          amountOrIdForApproval: "1",
-        });
+      });
+      if (!(basicFeeData instanceof Error)) {
         console.log(
-          "🚀 ~ file: App.tsx ~ line 259 ~ submit ~ approveTx",
-          approveTx
+          "🚀 ~ file: App.tsx ~ line 244 ~ submit ~ feeOracleData",
+          basicFeeData
         );
-        const depositTx = await (sygmaInstance as Sygma).deposit({
-          amount,
-          recipientAddress: address,
-          feeData: basicFeeData,
-        });
-        const depositEvent = await sygmaInstance!.getDepositEventFromReceipt(depositTx!)
-        const { depositNonce } = depositEvent.args;
-        setHomeDepositNonce(depositNonce.toNumber());
-        console.log("result of transfer", depositTx);
+        if (
+          window.confirm(
+            `Current fee: ${basicFeeData.calculatedRate} ${
+              basicFeeData.erc20TokenAddress === constants.AddressZero
+                ? "ETH"
+                : basicFeeData.erc20TokenAddress
+            }\n\nTotal(amount+fee): ${
+              parseFloat(amount) + parseFloat(basicFeeData.calculatedRate)
+            } tokens\n\nDo you really want to proceed?`
+          )
```

Comment:
This seems not right using `\n` in template strings as they have a [new line by using "enter"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings)

Reply:
Yes, template strings by 'enter' is supported, but it is against our linter rules and looks confusing in the code
<img width="610" alt="Screenshot 2023-01-12 at 20 09 40" src="https://user-images.githubusercontent.com/320313/212090652-72c261ea-9e58-4965-bf55-d412743d492b.png">

Reply:
how it looks in "production"?
It is possible that the babel transformer just transforms template stings into a regular one, in this case using `\n` will work correctly

Reply:
it looks equally as with `\n\n`.